src/test/java/AliasTest.java

```diff
@@ -0,0 +1,40 @@
+import io.restassured.RestAssured;
+import org.hamcrest.core.IsNull;
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.is;
+
+
+public class AliasTest extends AbstractFeatureServiceTest {
+
+    @Test
+    public void testFieldAlias() {
+        String path = request2path("Alias.json");
```

Comment:
It's fine to use this mechanism for generating test URLs but it was a carry-over from a conversion process I used when I built up the original test suite. You can drop the use of the JSON resource file and just do it like the tests in this one https://github.com/koopjs/koop-provider-marklogic/blob/master/src/test/java/WKTGeometry.java

It would look like this and you can get rid of the `Alias.json` resource
```suggestion
        String path = "/marklogic/{service}/FeatureServer/{layer}/query?resultRecordCount={resultRecordCount}& orderByFields={orderByFields}&returnGeometry={returnGeometry}";
                RestAssured
                .given()
                  .pathParam("GDeltGKG")
                  .pathParam("layer", 4)
                  .pathParam("resultRecordCount", 5)
                  .pathParam("orderByFields", "name ASC")
                  .pathParam("returnGeometry", true)
```